gemfirexd/shared/src/main/resources/com/pivotal/gemfirexd/internal/loc/toolsmessages.properties

```diff
@@ -614,8 +614,10 @@ AGENT_EXTRA_HELP=\
 \                 messages. Default is to use the hostname, or localhost\n\
 \                 in case hostname points to a local loopback address.\n\
 
-FS_WAITING_MESSAGE=Waiting for {0} (DiskId: {1}, Location: {2}) on: {3}
-FS_WAITING_MESSAGE_BUCKETS=Waiting for {0}, buckets {1} (DiskId: {2}, Location: {3}) on: {4}
+FS_WAITING_MESSAGE=Member disk state is not the most current. {0} is waiting for disk recovery from following members: {1}.\n\
+\Refer to log file for details
+FS_WAITING_MESSAGE_BUCKETS=Member disk state is not the most current. {0} is waiting for disk recovery buckets {1} from following members: {2}. \n\
```

Comment:
{0} is waiting for disk recovery <of> buckets {1}

notice 'of' there. What do you say?